framework/src/main/java/org/checkerframework/framework/stub/RemoveAnnotationsForInference.java

```diff
@@ -76,6 +94,26 @@ public static void main(String[] args) {
       System.err.println("Usage: provide one or more directory names to process");
       System.exit(1);
     }
+    // TODO: using plume-lib's options here would be better, but would add a dependency
+    // to the whole Checker Framework, which is undesirable. Move this program elsewhere
+    // (e.g., to a plume-lib project)?
+    if (args[0].contentEquals("-exclusionFile")) {
+      if (args.length < 3) {
+        System.err.println(
+            "Usage: -exclusionFile requires at least two more arguments: the path to "
```

Comment:
Those are not arguments of `-exclusionFile`; that is, `-exclusionFile` does not take two arguments.  Instead, have this remove the first two elements of `args` (oh, I see you do this), and move the test for a directory name to process below this.

Reply:
Sorry if this test was unclear; I've rewritten the error-message to be more specific.

However, I don't think I can move this test: its purpose is to guard the access to `args[1]` a few lines below, which requires an `@MinLen(2)` array. (Yes, I know we're not actually running the Index Checker on this code, but it's better to follow its rules, both for safety and in case we ever do want to run it here.)